doc/Create-rosa-hcp.md

```diff
@@ -2,68 +2,17 @@
-    2. Create the OpenID Connect (OIDC) configuration.
-
-       ```     
-       rosa create oidc-config --mode=auto
-       ```
-    3. Copy the OIDC config ID <OIDC_CONFIG_ID> provided in the ROSA CLI output. 
-      
-       The OIDC config ID needs to be provided later to create the ROSA HCP cluster.  
-       You can list the available OIDC config IDs using the following command:
-
-       ```     
-       rosa list oidc-config
-       ```
-
-   4. Create the required IAM operator roles.   
-   You must supply a prefix in <PREFIX_NAME> and replace the <OIDC_CONFIG_ID> with the OIDC config ID copied previously.
-
-      ```     
-       rosa create operator-roles --prefix <PREFIX_NAME> --oidc-config-id <OIDC_CONFIG_ID> --hosted-cp
-      ```
-     
-   5. Verify the IAM operator roles were created. 
-      ```     
-      rosa list operator-roles
-      ```
-
-4. Assuming you have an OpenShift cluster v4.16 or later running, install ACM (Advanced Cluster Management) operator v2.13 from the [operator hub](https://docs.redhat.com/en/documentation/red_hat_advanced_cluster_management_for_kubernetes/2.13/html/install/installing#installing-from-the-operatorhub) or using OpenShift Container Platform [CLI](https://docs.redhat.com/en/documentation/red_hat_advanced_cluster_management_for_kubernetes/2.13/html/install/installing#installing-from-the-cli)
-
-Notes: The steps for creating the AWS VPC and IAM roles will be declarative through CAPA custom resources next release.
-
-## Enable CAPI, CAPA and auto import 
-
-The MultiClusterEngine custom resource CAPI & CAPA features must be enabled once the ACM operator installation and pre-request steps have been completed. 
-
-1. Verify the default MultiClusterEngine CR has been created. 
+1. Complete the prerequisite actions listed in the [ROSA with HCP Prerequisites](https://docs.redhat.com/en/documentation/red_hat_openshift_service_on_aws/4/html/install_rosa_with_hcp_clusters/rosa-hcp-sts-creating-a-cluster-quickly#rosa-hcp-prereqs) section of the ROSA HCP documentation.
+  Note: The steps for creating the AWS VPC and IAM roles will be declarative through CAPA custom resources in a future release.
+
+2. Using an OpenShift cluster v4.16 or later running, install RHACM (Red Hat Advanced Cluster Management) operator v2.13 or later from the [installation documentation](https://docs.redhat.com/en/documentation/red_hat_advanced_cluster_management_for_kubernetes/2.13/html/install/installing#installing-from-the-operatorhub) or using OpenShift Container Platform [CLI](https://docs.redhat.com/en/documentation/red_hat_advanced_cluster_management_for_kubernetes/2.13/html/install/installing#installing-from-the-cli)
+
+## Enable CAPI, CAPA and auto import features in ACM (multicluster engine)
+
+The Multicluster Engine is automatically installed with RHACM. 
+The MultiClusterEngine custom resource contains both CAPI & CAPA feature flags which. These must be enabled only after the ACM operator installation is complete.
```

Comment:
Hi @berenss, Can we remove the ". These" here?

Reply:
yes good catch